### DISCUSSION TITLE
Can folder name be included in import name for subcomponets

### DISCUSSION THREAD

**Discussion by f3oall** [vuejs/language-tools#1914] [Q&A]
Imagine the following file structure:
```
- components
  - VForm
    - Input.vue (<VFormInput/>)
    - index.vue (<VForm/>)
```

Currently, I get these suggestions:
![image](https://user-images.githubusercontent.com/1751835/192923867-8aed2a26-c31b-4bbc-8ca5-fac5edf36da7.png)

So the question is, can it suggest `VFormInput' as a name instead of just 'Input'?

**Comment by johnsoncodehk** [comment]
IMO it will be overdesign, so we have no plan to implement.
